relnotes/stream-stuff.deprecation.md

```diff
@@ -0,0 +1,3 @@
+* `ocean.io.stream.Digester`, `ocean.io.stream.Endian`, `ocean.io.stream.Greedy`, `ocean.io.stream.Map`, `ocean.io.stream.Typed`, `ocean.io.stream.Utf`
+
+  Those module are unused within Sociomantic. They are also  barely tested and anticated, and thus have been deprecated.
```

Comment:
Just to make sure: when you make statements like this, you're doing searches over all repos, right?

Reply:
Typo "antiquated"

Reply:
> Just to make sure: when you make statements like this, you're doing searches over all repos, right?

Yep. In this case just looking for `user:githuborgname "ocean.io.stream.Typed"` (the quotes help) returns no result, whereas it would if it was imported anywhere

Reply:
> antiquated

Closer to French than I had expected :)

Reply:
I guess the pronunciation is further away, though ;) (anti-kwayted)